README.md

```diff
@@ -1 +1,301 @@
+  - `user:{emailid}`: An email address that represents a specific Google account. For example, alice@gmail.com or joe@example.com.
+  - `serviceAccount:{emailid}`: An email address that represents a service account. For example, my-other-app@appspot.gserviceaccount.com.
+  - `group:{emailid}`: An email address that represents a Google group. For example, admins@example.com.
+  - `domain:{domain}`: A G Suite domain (primary, instead of alias) name that represents all the users of that domain. For example, google.com or example.com.
+
+  Default is `[]`.
+
+- **`authoritative`**: _(Optional `bool`)_
+
+  Whether to exclusively set (authoritative mode) or add (non-authoritative/additive mode) members to the role.
+
+  Default is `false`.
+
+- **`condition`**: _(Optional `object(condition)`)_
+
+  An IAM Condition for a given binding.
+
+  Each `condition` object can have the following fields:
+
+  Example
+
+   ```hcl
+    condition = {
+      title       = "expires_after_2019_12_31"
+      description = "Expiring at midnight of 2019-12-31"
+      expression  = "request.time < timestamp(\"2020-01-01T00:00:00Z\")"
+  }
+   ```
+
+  - **`title`**: **_(Required `string`)_**
+
+    A title for the expression, i.e. a short string describing its purpose.
+
+  - **`expression`**: **_(Required `string`)_**
+
+    Textual representation of an expression in Common Expression Language syntax.
+
+  - **`description`**: _(Optional `string`)_
+
+    An optional description of the expression. This is a longer text which describes the expression, e.g. when hovered over it in a UI.
+
+- **`policy_bindings`**: _(Optional `list(policy_bindings)`)_
+
+  A list of IAM policy bindings.
+
+  Example
+
+  ```hcl
+  policy_bindings = [{
+    role    = "roles/viewer"
+    members = ["user:member@example.com"]
+  }]
+  ```
+
+  Each `policy_bindings` object accepts the following fields:
+
+  - **`role`**: **_(Required `string`)_**
+
+    The role that should be applied.
+
+  - **`members`**: **_(Required `string`)_**
+
+    Identities that will be granted the privilege in `role`.
+
+    Default is `var.members`.
+
+  - **`condition`**: _(Optional `object(condition)`)_
+
+    An IAM Condition for a given binding.
+
+    Example
+
+    ```hcl
+    condition = {
+      expression = "request.time < timestamp(\"2022-01-01T00:00:00Z\")"
+      title      = "expires_after_2021_12_31"
+    }
+    ```
+
+  A `condition` object accepts the following fields:
+
+  - **`expression`**: **_(Required `string`)_**
+
+    Textual representation of an expression in Common Expression Language syntax.
+
+  - **`title`**: **_(Required `string`)_**
+
+    A title for the expression, i.e. a short string describing its purpose.
+
+  - **`description`**: _(Optional `string`)_
+
+    An optional description of the expression. This is a longer text which describes the expression, e.g. when hovered over it in a UI.
```

Comment:
wrong indent

Reply:
i don't see it broken in visual cod
![Screenshot from 2021-11-04 12-13-06](https://user-images.githubusercontent.com/86191033/140296247-3e38970c-353f-4b22-816d-c3fca9a73d58.png)
e